Framework/HistogramData/inc/MantidHistogramData/FixedLengthVector.h

```diff
@@ -36,8 +36,6 @@ template <class T> class FixedLengthVector {
   FixedLengthVector(std::initializer_list<double> init) : m_data(init) {
     Validator<T>::checkValidity(m_data);
   }
-  FixedLengthVector(const FixedLengthVector &) = default;
```

Comment:
Could you help me understand why these need removing?

Reply:
Following rule of 1, since we do not define `FixedLengthVector &operator=(const FixedLengthVector &)`

Reply:
Ahh that's my fault, I mistakenly removed that operator 3 months ago then fixed it the wrong way :facepalm: